psydac/ddm/cart.py

```diff
@@ -1261,36 +1307,41 @@ def parent_starts( self ):
     def parent_ends( self ):
         return self._parent_ends
 
-    @property
-    def ranks_in_topo_plus( self ):
-        return self._ranks_in_topo_plus
-
     @property
     def local_group_minus( self ):
+        """ The MPI Group of the ranks defined in the minus patch"""
         return self._local_group_minus
 
     @property
     def local_group_plus( self ):
+        """ The MPI Group of the ranks defined in the plus patch"""
         return self._local_group_plus
 
     @property
     def local_comm_minus( self ):
+        """ The MPI intra-communicator defined in the minus patch"""
         return self._local_comm_minus
 
     @property
     def local_comm_plus( self ):
+        """ The MPI intra-communicator defined in the plus patch"""
         return self._local_comm_plus
 
     @property
     def local_comm( self ):
+        """ The communicator owned by the process it can be local_comm_minus or local_comm_plus."""
         return self._local_comm
```

Comment:
What do you mean with "communicator owned by the process"?

A process can belong to one or more communicators, but the contrary makes no sense to me.

Reply:
in our case, we split the MPI_COMM_WORLD to disjoint Intra sub-communicators, so a process belongs to one local intra- communicator

Reply:
Then you should change "The communicator owned by the process" with "The sub-communicator to which the process belongs", and also add a comma "," before "it can be"

Reply:
done in [eab4eb9](https://github.com/pyccel/psydac/pull/213/commits/eab4eb9035ec7950199e1b03e2eaf7652bea8362)